src/schemas/generator/README.md

```diff
@@ -0,0 +1,25 @@
+# Generator migration
+
+Migrations are needed when changing the structure of existing generators file such as renaming or deleting keys.
+
+If you're simply adding new options to the generator file, a migration may not be needed. In this case, extend the latest schema available.
+
+## Creating a new migration
+
+For when a new migration is needed:
+
+1. Create a directory for the new version (e.g. `/v2`).
+2. Copy files from the previous schema into this directory and make appropriate changes.
```

Comment:
While out of scope for this PR, I think we should explore autogenerating new schema versions in future - instead of manually creating a copy of the `vN` folder and its content, we'd be able to run something like `whatever-code-gen-is-trendy-rn studio:generator` and have it do it automatically.

Reply:
We can explore that separately and probably do the same for settings too! 👍